src/services/formatting/smartIndenter.ts

```diff
@@ -322,6 +322,26 @@ namespace ts.formatting {
             return false;
         }
 
+        export function childStartsInlineWithPreviousObject(parent: Node, child: TextRangeWithKind, childStartLine: number, sourceFile: SourceFileLike): boolean {

```

Comment:
Is the previous thing necessarily an object? If not, I’d probably call this `argumentStartsOnSameLineAsPreviousArgument` since this function is specific to call expressions.